src/Splunk.ModularInputs/Splunk/ModularInputs/ModularInput.cs

```diff
@@ -297,7 +297,9 @@ private static bool IsValidateArguments(string[] args, DebuggerAttachPoints atta
                     try
                     {
                         inputDoc = await stdin.ReadToEndAsync();
-                        writer.LogAsync(Severity.Debug, inputDoc).Wait();
+                        inputDoc = inputDoc.Replace(Environment.NewLine, " |");
```

Comment:
Sounds ok to me especially now that we're not in a rush :-)

On Tue Feb 03 2015 at 11:19:16 PM Shakeel Mohamed <notifications@github.com>
wrote:

> In src/Splunk.ModularInputs/Splunk/ModularInputs/ModularInput.cs
> <https://github.com/splunk/splunk-sdk-csharp-pcl/pull/32#discussion_r24066594>
> :
>
> > @@ -297,7 +297,9 @@ private static bool IsValidateArguments(string[] args, DebuggerAttachPoints atta
> >                      try
> >                      {
> >                          inputDoc = await stdin.ReadToEndAsync();
> > -                        writer.LogAsync(Severity.Debug, inputDoc).Wait();
> > +                        inputDoc = inputDoc.Replace(Environment.NewLine, " |");
>
> Yeah, so this could become something like inputDoc =
> Utils.ReplaceNewLinesWithPipes(inputDoc);
>
> Up to you :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/splunk/splunk-sdk-csharp-pcl/pull/32/files#r24066594>.
>